tools/datafile-generators/h1linkdef.h

```diff
@@ -0,0 +1,16 @@
+//
+//  LinkDef.h
+//
+//
+//  Created by Simon Leisibach on 25.09.19.
+//
+
+#ifdef __CINT__
+
+#pragma link off all globals;
```

Comment:
The link off are no longer needed. (default was switch a long time ago :) )

Reply:
Thanks!!! I didn't know!